### PR TITLE
fix default values for ENV time params

### PR DESCRIPTION
**Pull request by friedolino78**
problem with seems to be solved.

### CODE REVIEW

File: src/Params/EnvelopeParams.cpp [hunk 1]
```diff
@@ -324,17 +324,17 @@ void EnvelopeParams::init(zyn::consumer_location_t _loc)
...
-        case ad_voice_fm_amp:  ADSRinit(80, 90, 127, 100); break;
-        case sub_freq:         ASRinit(30, 0.025, 64, 0.5); break;
-        case sub_bandwidth:    ASRinit_bw(100, 0.97, 64, 0.5); break;
+        case sub_filter:       ADSRinit_filter(64, 0.127f, 64, 0.970f, 0.499f, 64); break;
+        case ad_voice_amp:     ADSRinit_dB(0, 6.978f, 127, 6.978f); break;
+        case ad_voice_freq:    ASRinit(30, 0.127f, 64, 0.499f); break;
+        case ad_voice_filter:  ADSRinit_filter(90, 0.970f, 40, 0.970f, 0.009f, 40); break;
+        case ad_voice_fm_freq: ASRinit(20, 3.620f, 40, 1.876f); break;
+        case ad_voice_fm_amp:  ADSRinit(1.876f, 3.620f, 127, 6.978f); break;
+        case sub_freq:         ASRinit(0.061f, 0.254f, 64, 0.499f); break;
```
Comment (JohannesLorenz) [inline src/Params/EnvelopeParams.cpp:336]: The first parameter expects a `char` here.

File: src/Params/EnvelopeParams.cpp [hunk 2]
```diff
@@ -324,17 +324,17 @@ void EnvelopeParams::init(zyn::consumer_location_t _loc)
...
-        case sub_filter:       ADSRinit_filter(64, 0.01, 64, 0.025, 0.05, 64); break;
-        case ad_voice_amp:     ADSRinit_dB(0, 7.0, 127, 7.0); break;
-        case ad_voice_freq:    ASRinit(30, 0.01, 64, 0.5); break;
-        case ad_voice_filter:  ADSRinit_filter(90, 0.025, 40, 0.025, 0.009, 40); break;
-        case ad_voice_fm_freq: ASRinit(20, 3.62, 40, 1.876); break;
-        case ad_voice_fm_amp:  ADSRinit(80, 90, 127, 100); break;
-        case sub_freq:         ASRinit(30, 0.025, 64, 0.5); break;
-        case sub_bandwidth:    ASRinit_bw(100, 0.97, 64, 0.5); break;
+        case sub_filter:       ADSRinit_filter(64, 0.127f, 64, 0.970f, 0.499f, 64); break;
+        case ad_voice_amp:     ADSRinit_dB(0, 6.978f, 127, 6.978f); break;
```
Comment (JohannesLorenz) [inline src/Params/EnvelopeParams.cpp:331]: Just to make clear for the reader that it's a float:
```suggestion
        case ad_voice_amp:     ADSRinit_dB(0.f, 6.978f, 127, 6.978f); break;
```

**Review by JohannesLorenz — CHANGES_REQUESTED** [review]
Only 2 comments. Please also change the "no used" to "not used".

Invite to re-review when you're finished.

File: src/Params/EnvelopeParams.cpp (hunk 1 above)
Comment (friedolino78) [reply src/Params/EnvelopeParams.cpp]: you are right.

**Review by JohannesLorenz — APPROVED** [review]
All good. You can merge it after CI has passed.

**Pull request merged**